s/exec_aten::/executorch::aten::/ for extension/**/*.h

dbort opened the pull request:
Summary: Migrate all extension headers to use the new aten namespace, so that they act as good examples for users. The .cpp code can migrate later.

Differential Revision: D64079593

malfet reviewed (approved):
I trust that you did it with some sort of regex, but wouldn't it be a BC-break change?

dbort committed:
s/exec_aten::/executorch::aten::/ for extension/**/*.h (#6032)

Summary:

Migrate all extension headers to use the new aten namespace, so that they act as good examples for users. The .cpp code can migrate later.

Reviewed By: lucylq

Differential Revision: D64079593

dbort commented:
@pytorchbot cherry-pick --onto release/0.4 -c fixnewfeature
